_posts/2024-04-30-A-comprehensive-guide-to-setup-and-connect-self-managed-dashboards-with-Amazon-OpenSearch-domain.md

```diff
@@ -0,0 +1,275 @@
+---
+layout: post
+title:  "A comprehensive guide to setting up and connecting self-managed OpenSearch Dashboards with an Amazon OpenSearch Service domain"
+authors:
+ - mkbn
+date: 2024-04-30
+categories:
+ - technical-post
+meta_keywords: OpenSearch dashboards, Amazon OpenSearch service, self-managed, downtime issues, Mitigation
```

Comment:
Please update the meta with the following: 

**meta_keywords:** OpenSearch Dashboards, Amazon OpenSearch Service, self-managed dashboards, blue/green deployments

**meta_description**: Read this guide to blue/green deployments to learn how you can set up self-managed dashboards in Amazon OpenSearch, including no authentication, basic authentication, and SAML authentication.